gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -303,30 +306,52 @@ def _get_freq_by_pop(
             )
             for pop in pops
         ]
-    ).to_pandas()
-    df["cmh"] = df.apply(lambda x: StratifiedTable(x.an).test_null_odds(), axis=1)
+    )
+    _ht = _ht.add_index(name="tmp_idx")
+    _ht = _ht.annotate(tmp_idx=hl.int32(_ht.tmp_idx))
+    tmp_path = hl.utils.new_temp_file("cmh_test", "parquet")
+
+    # Export one compressed Parquet file per partition.
+    df = _ht.to_spark().write.option("compression", "zstd")
+    df.mode("overwrite").parquet(tmp_path)
+    df = pd.read_parquet(tmp_path)
+    df["cmh"] = df.apply(
+        lambda x: StratifiedTable(
+            [list([list(a) for a in p]) for p in x.an]
+        ).test_null_odds(),
+        axis=1,
+    )
```

Comment:
Maybe just a note what the `[list([list...` is doing, from conversations before this and docs I think its just building that 2x2 matrix but just want to confirm and note for future reference.